docs/intro/using.rst

```diff
@@ -151,6 +153,12 @@ with large numbers of variables.
         hybrid.EnergyImpactDecomposer(size=50, rolling_history=0.15)
     )
 
+    # Helper function to merge subsamples in place
+    def merge_substates(_, substates):
```

Comment:
What's the `_,` needed for here?

Reply:
It means the first argument is irrelevant here (it's a reference to `Lambda` instance we don't use in the code).